treatment-service/appcontext/appcontext.go

```diff
@@ -92,18 +93,17 @@ func NewAppContext(cfg *config.Config) (*AppContext, error) {
 		return nil, err
 	}
 
-	log.Println("Initializing message queue subscriber...")
 	var messageQueueService messagequeue.MessageQueueService
-	switch cfg.MessageQueueConfig.Kind {
-	case common_mq_config.NoopMQ:
+	if cfg.PollerConfig.Enabled || cfg.MessageQueueConfig.Kind == common_mq_config.NoopMQ {
```

Comment:
I think we can just leave the polling config and message queue config as independent configuration i.e. whether the poller is set up or not doesn't have an impact on whether the message queue is configured or not. In other words, we can have the poller enabled BUT with a valid message queue (pubsub in this case) enabled also.

That is to say, let's remove the checks on the poller config here.